mlrun/projects/operations.py

```diff
@@ -77,6 +77,7 @@ def run_function(
     auto_build: Optional[bool] = None,
     schedule: Union[str, mlrun.common.schemas.ScheduleCronTrigger] = None,
     artifact_path: Optional[str] = None,
+    output_path: Optional[str] = None,
```

Comment:
When adding new arguments to client-facing functions, add them as the last argument.
If not, if users call this function without explicit keyword arguments it will fail.